app/controllers/concerns/api_responders.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module ApiResponders
+  extend ActiveSupport::Concern
+
+  private
+
+    def respond_with_error(message, status = :unprocessable_entity, context = {})
+      is_message_array = message.is_a?(Array)
+      error_message = is_message_array ? message.to_sentence : message
+      render status: status, json: { error: error_message }.merge(context)
+    end
+
+    def respond_with_success(message, status = :ok, context = {})
+      render status: status, json: { notice: message }.merge(context)
+    end
+
+    def respond_with_json(status = :ok, context = {})
+      render status: status, json: context
+    end
```

Comment:
Keep it aligned with granite.

Reply:
@yedhink As I mentioned  in https://github.com/bigbinary/bigbinary-website/pull/1931#discussion_r804704875. we should update Granite. 

Please let me know your thoughts.